static/app/components/modals/widgetBuilder/addToDashboardModal.tsx

```diff
@@ -14,13 +21,33 @@ import {t, tct} from 'sentry/locale';
 import space from 'sentry/styles/space';
 import {Organization, PageFilters, SelectValue} from 'sentry/types';
 import useApi from 'sentry/utils/useApi';
-import {DashboardListItem, MAX_WIDGETS, Widget} from 'sentry/views/dashboardsV2/types';
+import {
+  DashboardListItem,
+  DisplayType,
+  MAX_WIDGETS,
+  Widget,
+} from 'sentry/views/dashboardsV2/types';
 import WidgetCard from 'sentry/views/dashboardsV2/widgetCard';
 
+type WidgetAsQueryParams = Query & {
+  defaultTableColumns: string[];
+  defaultTitle: string;
+  defaultWidgetQuery: string;
+  displayType: DisplayType;
+  environment: string[];
+  project: number[];
+  source: string;
+  end?: unknown;
+  start?: unknown;
+  statsPeriod?: unknown;
```

Comment:
```suggestion
  statsPeriod?: string | null,
  start?: DateString,
  end?: DateString
```
usually these props have the above types

Reply:
Thank you! I meant to type them after with the other props but forgot.